crm.py

```diff
@@ -0,0 +1,61 @@
+# Jede Änderung an der Excel soll natürlich gespeichert werden!
+# Drei Verkäufer: Markus, Jürgen und Jasmin
+# Zwei Status/Stages: Offen (Cold Call), Abgeschlossen (Verkauf, Follow Up)
+import openpyxl
+from openpyxl.chart import Reference, Series
+# Schreibe eine Funktion die neuen Kunde mit allen Wert ins CRM aufnimmt. Werte sollen über den Argparse hinzugefügt werden können. Und füge min. 10 Kunden mit verschiedenen Werten über diesen Aufruf hinzu in die Excel. 
+import argparse
+parser = argparse.ArgumentParser()
+parser.add_argument("-N", "--Name", dest="crm.xlsx", help="wie lautet der Name von dem Kunden", required=True)
```

Comment:
```suggestion
parser.add_argument("-N", "--Name", dest="name", help="wie lautet der Name von dem Kunden", required=True)
```

dest ist immer die Variable die du über args.name abrufen kannst. Daher muss jedes Argument ein anderes dest haben. Gilt bis Zeile 16